modules/launcher/src/main/java/org/mule/module/launcher/log4j2/MuleLoggerContext.java

```diff
@@ -73,13 +80,24 @@
             applicationClassloader = false;
             artifactName = null;
         }
+
+        Collection<Field> candidateFields = getAllFields(LoggerContext.class, withName("loggers"));
```

Comment:
the loggers field is private in the superclass. This allows to override the #getLogger(final String name, final MessageFactory messageFactory) method.